### PR TITLE
Update BN/GMT/NAT/BSC/osu! Alumni listings, `Staff log` and `ko.yaml`

### PR DESCRIPTION
**Pull request by aceticke**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

As always input is encouraged for the GMT pages as the majority of new additions have placeholders until their language can be confirmed, furthermore if they are assigned to a specific language's chat let me know so I can add them.

### CODE REVIEW

File: wiki/People/Nomination_Assessment_Team/es.md [hunk 1]
```diff
@@ -81,50 +81,50 @@ La [página del grupo Nomination Assessment Team](https://osu.ppy.sh/groups/7) e
...
+| ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | Portugués | Evaluación |
+| ::{ flag=AU }:: [elicz1](https://osu.ppy.sh/users/8039342) |  | Evaluación |
+| ::{ flag=CN }:: [Firika](https://osu.ppy.sh/users/9590557) | Chino | Evaluación |
+| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | Alemán | Evaluación, estructural |
+| ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | Sueco | Estructural[^task-naxess] |
+| ::{ flag=GB }:: [nexusqi](https://osu.ppy.sh/users/13822800) |  | Evaluación |
+| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) |  | Estructural[^task-pishifat] |
+| ::{ flag=US }:: [StarCastler](https://osu.ppy.sh/users/12402453) |  | Evaluación, estructural |
+| ::{ flag=AT }:: [Stixy](https://osu.ppy.sh/users/9000308) | Alemán, serbio | Evaluación |
+| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) |  | Evaluación |
```
Comment (FazlyMR) [inline wiki/People/Nomination_Assessment_Team/es.md:98]:
```suggestion
| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) |  | Evaluación[^task-uberfazz] |
```

File: wiki/People/Nomination_Assessment_Team/fr.md [hunk 2]
```diff
@@ -80,50 +80,50 @@ La [page du groupe de la Nomination Assessment Team](https://osu.ppy.sh/groups/7
...
+| ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | Portugais | Evaluation |
+| ::{ flag=AU }:: [elicz1](https://osu.ppy.sh/users/8039342) |  | Evaluation |
+| ::{ flag=CN }:: [Firika](https://osu.ppy.sh/users/9590557) | Chinois | Evaluation |
+| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | Allemand | Evaluation, structural |
+| ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | Suédois | Structural[^task-naxess] |
+| ::{ flag=GB }:: [nexusqi](https://osu.ppy.sh/users/13822800) |  | Evaluation |
+| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) |  | Structural[^task-pishifat] |
+| ::{ flag=US }:: [StarCastler](https://osu.ppy.sh/users/12402453) |  | Evaluation, structural |
+| ::{ flag=AT }:: [Stixy](https://osu.ppy.sh/users/9000308) | Allemand, serbe | Evaluation |
+| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) |  | Evaluation |
```
Comment (FazlyMR) [inline wiki/People/Nomination_Assessment_Team/fr.md:97]:
```suggestion
| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) |  | Evaluation[^task-uberfazz] |
```

File: wiki/People/Nomination_Assessment_Team/zh.md [hunk 3]
```diff
@@ -84,50 +84,50 @@ tags:
...
+| ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | 葡萄牙语 | Evaluation |
+| ::{ flag=AU }:: [elicz1](https://osu.ppy.sh/users/8039342) |  | Evaluation |
+| ::{ flag=CN }:: [Firika](https://osu.ppy.sh/users/9590557) | 汉语 | Evaluation |
+| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) | 德语 | Evaluation，structural |
+| ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | 瑞典语 | Structural[^task-naxess] |
+| ::{ flag=GB }:: [nexusqi](https://osu.ppy.sh/users/13822800) |  | Evaluation |
+| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) |  | Structural[^task-pishifat] |
+| ::{ flag=US }:: [StarCastler](https://osu.ppy.sh/users/12402453) |  | Evaluation，structural |
+| ::{ flag=AT }:: [Stixy](https://osu.ppy.sh/users/9000308) | 德语，塞尔维亚语 | Evaluation |
+| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) |  | Evaluation |
```
Comment (FazlyMR) [inline wiki/People/Nomination_Assessment_Team/zh.md:101]:
```suggestion
| ::{ flag=US }:: [UberFazz](https://osu.ppy.sh/users/8646059) |  | Evaluation[^task-uberfazz] |
```

**Review by FazlyMR — COMMENTED** [review]
This should resolve the remark `remark-link:no-unused-definitions` errors. If the `[^task-uberfazz]` definition isn't needed anymore then you do have the option to remove any mentions of the definition, that would also resolve the error.

File: wiki/People/Nomination_Assessment_Team/es.md (hunk 1 above)
Comment (aceticke) [reply wiki/People/Nomination_Assessment_Team/es.md:98]: No longer in NAT, so I removed the definition for now

**Review by TicClick — CHANGES_REQUESTED** [review]
- spoken languages missing for new GMTs
- as well as areas of activity
- and rows in the "grouped by language/channel" table

**Comment by aceticke** [comment]
> * spoken languages missing for new GMTs
> * as well as areas of activity
> * and rows in the "grouped by language/channel" table

will need further assistance with this from the gmt group or nat

File: wiki/People/Global_Moderation_Team/en.md [hunk 4]
```diff
@@ -141,8 +149,8 @@ The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all o
...
-| `#finnish` | [Suomi](https://osu.ppy.sh/community/forums/24) | ::{ flag=FI }:: [Laurakko](https://osu.ppy.sh/users/7253731) |
-| `#french` | [Français](https://osu.ppy.sh/community/forums/34) | ::{ flag=FR }:: [Imakuri](https://osu.ppy.sh/users/6100837), ::{ flag=FR }:: [Nozhomi](https://osu.ppy.sh/users/2716981), ::{ flag=FR }:: [Pachiru](https://osu.ppy.sh/users/2850983), ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) |
+| `#finnish` | [Suomi](https://osu.ppy.sh/community/forums/24) | ::{ flag=FI }:: [Flutteh](https://osu.ppy.sh/users/5042507), ::{ flag=FI }:: [Laurakko](https://osu.ppy.sh/users/7253731) |
+| `#french` | [Français](https://osu.ppy.sh/community/forums/34) | ::{ flag=FR }:: [Ganondorf](https://osu.ppy.sh/users/10660738), ::{ flag=FR }:: [Imakuri](https://osu.ppy.sh/users/6100837), ::{ flag=FR }:: [Nozhomi](https://osu.ppy.sh/users/2716981), ::{ flag=FR }:: [Pachiru](https://osu.ppy.sh/users/2850983), ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005) |
```
Comment (hikikuman) [inline wiki/People/Global_Moderation_Team/en.md:153]: you forgor to remove pachiru from the list. please also check the other file

**Review by hikikuman — CHANGES_REQUESTED** [review]

File: wiki/People/Global_Moderation_Team/en.md (hunk 4 above)
Comment (aceticke) [reply wiki/People/Global_Moderation_Team/en.md:153]: oopsie daisy

**Review by TicClick — APPROVED** [review]
looks good to me, let's get this out

**Pull request merged**